stream_alert/rule_processor/firehose.py

```diff
@@ -49,7 +49,10 @@ class StreamAlertFirehose(object):
     def __init__(self, region, firehose_config, log_sources):
         self._firehose_client = boto3.client('firehose', region_name=region)
         # Expand enabled logs into specific subtypes
-        self._enabled_logs = self._load_enabled_log_sources(firehose_config, log_sources)
+
+        self._enabled_logs = []
+        if firehose_config:
```

Comment:
what prompted this change? the code cannot run without the loaded config, so it may as well just return or raise

Reply:
We can talk about this in person but this was added because the default global config does not contain a `firehose` config.. yet this class is very tightly coupled to the table creation. Would be better alternative be to have a default `firehose` config that has an empty list of `enabled_logs`?

Reply:
that's a reasonable workaround